### PR TITLE
Tbl/additional earning

### LINKED ISSUE #33: tbl_addtnl _earnings
Create a TABLE/MODEL/ENTITY on database and CRUD for tbl_addtnl _earnings

Reference:
https://github.com/TempName-1/rdi-payroll-system/blob/dev/backend/README.md

### PR DESCRIPTION
**Pull request by yell66w**
# Description
Galing kay @CMHeat 

Fixes #33 

# Type of change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

# Screenshots (if appropriate)

Add screenshots that could help the reviewer.

# Checklist:

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by yell66w** [comment]
@franzegos patest nga kung nagana

File: backend/src/models/additional_earnings.model.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            },
+            date: {
+                type: DataTypes.DATE(),
+                allowNull: false,
+            },
+            total: {
+                type: DataTypes.DECIMAL(19, 4),
+                allowNull: false,
+            },
+        },
```
Comment (franzegos) [inline backend/src/models/additional_earnings.model.js:36]: Kulang ng remarks sirrr

Comment (yell66w) [reply backend/src/models/additional_earnings.model.js:36]: @CMHeat madadagdagan mo gn remarks lods?

**Comment by franzegos** [comment]
Then if may remarks na ganto sya:
image

1. yung date hindi na sya input, automated na sya kagaya dun sa "date_created"
2. yung total is hindi rin input, i-add nalnag lahat ng value nung reimbursement, additional, tsaka others.

**Pull request merged**